sdcm/sct_runner.py

```diff
@@ -104,10 +107,10 @@ def install_prereqs(self, public_ip):
             # Jenkins pipelines run /bin/sh for some reason
             unlink /bin/sh
             ln -s /bin/bash /bin/sh
-
+            echo "cd ~/home/ubuntu" >> /home/ubuntu/.bashrc
         """)
         remoter = self.get_remoter(host=public_ip)
-        result = remoter.run(f"sudo bash -cxe '{prereqs_script}'", ignore_status=True)
+        result = remoter.run(f"sudo bash -cxe '{prereqs_script}'", ignore_status=False)
```

Comment:
This one looks unrelated to addition of "kubectl".
If we need it, then better to make it be separate commit in the same PR with description why it is done.

Reply:
Not needed and now reverted to `True`